tests/loggers/test_base.py

```diff
@@ -139,259 +64,6 @@ def after_save_checkpoint(self, checkpoint_callback):
-    )
-    trainer.fit(model)
-
-    if logger:
-        log_hyperparams_mock.assert_called()
-    else:
-        log_hyperparams_mock.assert_not_called()
-
-
-@patch("pytorch_lightning.loggers.tensorboard.TensorBoardLogger.log_hyperparams")
-def test_log_hyperparams_key_collision(log_hyperparams_mock, tmpdir):
-    class TestModel(BoringModel):
-        def __init__(self, hparams: Dict[str, Any]) -> None:
-            super().__init__()
-            self.save_hyperparameters(hparams)
-
-    class TestDataModule(BoringDataModule):
-        def __init__(self, hparams: Dict[str, Any]) -> None:
-            super().__init__()
-            self.save_hyperparameters(hparams)
-
-    class _Test:
-        ...
-
-    same_params = {1: 1, "2": 2, "three": 3.0, "test": _Test(), "4": torch.tensor(4)}
-    model = TestModel(same_params)
-    dm = TestDataModule(same_params)
-
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        max_epochs=1,
-        limit_train_batches=0.1,
-        limit_val_batches=0.1,
-        num_sanity_val_steps=0,
-        enable_checkpointing=False,
-        enable_progress_bar=False,
-        enable_model_summary=False,
-    )
-    # there should be no exceptions raised for the same key/value pair in the hparams of both
-    # the lightning module and data module
-    trainer.fit(model)
-
-    obj_params = deepcopy(same_params)
-    obj_params["test"] = _Test()
-    model = TestModel(same_params)
-    dm = TestDataModule(obj_params)
-    trainer.fit(model)
-
-    diff_params = deepcopy(same_params)
-    diff_params.update({1: 0, "test": _Test()})
-    model = TestModel(same_params)
-    dm = TestDataModule(diff_params)
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        max_epochs=1,
-        limit_train_batches=0.1,
-        limit_val_batches=0.1,
-        num_sanity_val_steps=0,
-        enable_checkpointing=False,
-        enable_progress_bar=False,
-        enable_model_summary=False,
-    )
-    with pytest.raises(MisconfigurationException, match="Error while merging hparams"):
-        trainer.fit(model, dm)
-
-    tensor_params = deepcopy(same_params)
-    tensor_params.update({"4": torch.tensor(3)})
-    model = TestModel(same_params)
-    dm = TestDataModule(tensor_params)
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        max_epochs=1,
-        limit_train_batches=0.1,
-        limit_val_batches=0.1,
-        num_sanity_val_steps=0,
-        enable_checkpointing=False,
-        enable_progress_bar=False,
-        enable_model_summary=False,
-    )
-    with pytest.raises(MisconfigurationException, match="Error while merging hparams"):
-        trainer.fit(model, dm)
+def test_lightning_logger_base_deprecation_warning():
+    with pytest.deprecated_call(match="The `pytorch_lightning.loggers.base.LightningLoggerBase` is deprecated."):
+        CustomDeprecatedLogger()
```

Comment:
can we move this test to the file `tests/deprecated_api/test_remove_1-9.py`. Thanks :)